### PR TITLE
Task 342, Update DG, UG and PPP

### LINKED ISSUE #235: [PE-D][Tester D] Output of documentation does not match actual output
When testing the `show_sales`, the documentation gives the standard output starting with:

However, that is now how it is displayed in the standard output. 

This could be due to a formatting error on the markdown of the GitHub pages or due to the user guide not being updated to the most recent version.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: farissirraj/ped#7

### LINKED ISSUE #241: [PE-D][Tester F] Formatting error in UG show_sales
**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>

Format from ### Show sales to ### Returning to the previous day seems off

-------------
Labels: `type.DocumentationBug` `severity.High`
original: limyuhching/ped#5

### LINKED ISSUE #242: [PE-D][Tester F] Duplicate line in UG
**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>

"list followed by list_ingredients 1 lists the ingredients of the 1st dish on the menu" is duplicated

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: limyuhching/ped#6

### LINKED ISSUE #256: [PE-D][Tester A] Documentation for add_order
**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
The format for the documentation for add_order and show_sales is quite confusing and messy. Perhaps there is some formatting issue?

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: BenedictChannn/ped#9

### LINKED ISSUE #342: Edit DG, PPP and UG to align with updated code for list_total_sales, list_ingredients and necessary pantry stock functions.

### PR DESCRIPTION
**Pull request by NaychiMin**
Fixes #342 
Fixes #256
Fixes #242
Fixes #241
Fixes #235

### CODE REVIEW

File: docs/images/sequence/Pantry_CalculateMaxDish.png
Comment (ziyi105) [inline docs/images/sequence/Pantry_CalculateMaxDish.png:1]: Perhaps in step 3 you could provide more info by using dishName:String?

Comment (ziyi105) [inline docs/images/sequence/Pantry_CalculateMaxDish.png:1]: is there any reason why there is a ":" before dishIngredients in step 5?

File: docs/images/sequence/Pantry_IsDishCooked.png
Comment (ziyi105) [inline docs/images/sequence/Pantry_IsDishCooked.png:1]: is there any reason why there is ":" before usedIngredientFromStock in step 9

Comment (ziyi105) [inline docs/images/sequence/Pantry_IsDishCooked.png:1]: Perhaps you should use the primitive "boolean" instead of "Boolean" here?

**Review by ziyi105 — APPROVED** [review]
Very detailed explanations, good job!

**Review by DextheChik3n — COMMENTED** [review]
I noticed in your sequence diagrams that the classes are missing a colon ':' before the class name

**Pull request merged**